deploy/contracts/docs/state/contract-state-detail.md

```
@@ -0,0 +1,972 @@
+    participant CU2 as Compute Unit 2
+    participant SU as Scheduler Unit
+    participant AW as Arweave
+
+    Client->>CU1: 状態取得要求
+    Client->>CU2: 状態取得要求（競争）
+
+    par 競争的解決
+        CU1->>SU: メッセージ履歴取得
+        SU->>CU1: 順序付きメッセージ
+        CU1->>AW: WASM/データ参照
+        AW->>CU1: 必要データ
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)